#Parameters for Relayboard

#Device on which relayboard is attached
ComPort: /dev/ttyS7
relayboard_timeout: 2.0

#Define the version of the protocol used by the Relayboard
protocol_version: 2
